src/main.rs

```diff
@@ -366,17 +366,13 @@ fn inner_main() -> Result<ExitCode> {
         }
     } else {
         let messages = if cli.files == vec![PathBuf::from("-")] {
-            if cli.fix {
-                eprintln!("Warning: --fix is not enabled when reading from stdin.");
-            }
-
             let filename = cli.stdin_filename.unwrap_or_else(|| "-".to_string());
             let path = Path::new(&filename);
-            run_once_stdin(&settings, path)?
+            run_once_stdin(&settings, path, cli.fix)?
         } else {
             run_once(&cli.files, &settings, !cli.no_cache, cli.fix)?
         };
-        if !cli.quiet {
+        if !cli.quiet && !cli.fix {
```

Comment:
I think we only want to _not_ print here if it came from `stdin` (since in that case, the contents went to `stdout`). Should we move this check + block into the `if` and `else` cases above?

Reply:
Oh I don't even know how I missed this lol Will address this and the other feedback later today!

Reply:
Awesome, thanks! No rush of course :)